.teamcity/src/main/kotlin/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -42,6 +42,14 @@ val m2CleanScriptWindows = """
     )
 """.trimIndent()
 
+val cleanAndroidUserHomeScriptUnixLike = """
+    rm -rf ~/.android
```

Comment:
Shouldn't we use `%teamcity.agent.jvm.user.home%` here as well?

Reply:
yes, you're right.

Reply:
I created https://github.com/gradle/gradle-private/issues/3380